### DISCUSSION TITLE
Auto Field Selected From Other Modules

### DISCUSSION THREAD

**Discussion by VictorTWB** [YetiForceCompany/YetiForceCRM#17215] [How to]
Hi Everyone, I have created a module "Repair Works" with a 1:M Field (Vehicle Number), upon selecting the vehicle field, I would like the field (Owner) to be auto filled with the vehicle owner of the vehicle. Has anyone done it before?

[https://crm.tmd.sg/q1.jpg](url)
[https://crm.tmd.sg/q2.jpg](url)

TIA !

**Comment by vovpff** [comment]
Thid can be done by self written handler

**Reply by VictorTWB** [reply]
Thank you for the reply. Is there any link or reference to refer to?

**Reply by vovpff** [reply]
Pls forget about handler. Check for file module_hierarchy.php. This mechanism can help you to call data from related module
